packages/bitski-provider/src/utils/transaction.ts

```diff
@@ -79,6 +80,8 @@ const createContext = <T extends EthSignMethod>(
       if (params && params?.length > 0) {
         return {
           from: params[0] as string,
+          chainId: parseInt(chain.chainId, 16),
```

Comment:
@ptescher - any reason to not do this? I was only able to test on v4 transactions I found in the wild, so unsure if this would affect anything for the other versions. Currently having trouble getting Metamask's test dapp to recognize Gorli for testing their sign typed functions.